sbpy/activity/dust.py

```diff
@@ -19,15 +22,16 @@
 import astropy.units as u
 
 try:
-    from astropy.utils.misc import InheritDocstrings
+    import scipy
+    from scipy.interpolate import splrep, splev
 except ImportError:
```

Comment:
Is this except block necessary?  `scipy` as a variable in this case is not used.